Formula/tmuxinator.rb

```diff
@@ -0,0 +1,44 @@
+class Tmuxinator < Formula
+  desc "Manage complex tmux sessions easily"
+  homepage "https://github.com/tmuxinator/tmuxinator"
+  url "https://github.com/tmuxinator/tmuxinator/archive/v0.15.0.tar.gz"
+  sha256 "e5c121126aebe3afc758c0561b8ef05508712a799d3821453063b87445806ed4"
+  depends_on "tmux"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "tmuxinator.gemspec"
```

Comment:
This creates an issue that the dependencies are not versioned. If you look at the `travis.rb` formula you can see that the gems are listed as resources which the gem is then build from and something similar would be needed here to make reproducible builds. Can you arrange that ?